lua/clangd_extensions/type_hierarchy.lua

```diff
@@ -54,82 +53,91 @@ local function format_tree(node, visited, result, padding, type_to_location)
     return result
 end
 
+local function buf_request_method(method, params, handler, bufnr)
+    local clients = vim.lsp.get_clients({ bufnr = bufnr, method = method })
+    for _, client in pairs(clients) do
+        client:request(method, params, handler, bufnr)
+    end
+end
+
```

Comment:
this function is duplicated 5 times, maybe this can go into a `utils.lua`

Reply:
My bad: https://github.com/p00f/clangd_extensions.nvim/pull/75#discussion_r2249851974


> this function is duplicated 5 times, maybe this can go into a `utils.lua`

Yeah, that's what I thought, but went for the lazy path. Editing now.